src/utils.js

```diff
@@ -166,7 +177,7 @@ export function isFunction(target) {
  * @return {boolean}
  * @private
  */
-export function isString(target) {
+export function isString(target: *): boolean {
```

Comment:
does the `*` mean `any`? or `mixed`? or just letting it infer? if the last one then is it needed?